test/backend_test.in.cpp

```diff
@@ -417,6 +417,56 @@ TEST(${BACKEND_NAME}, concat_vector)
     EXPECT_EQ((vector<float>{2, 4, 8, 16, 1, 2, 4, 8, 16, 32, 18, 19}), read_vector<float>(result));
 }
 
+TEST(${BACKEND_NAME}, concat_4d_tensor)
```

Comment:
In the future please add appropriate backend skips for new tests if other tests of the same op also skip those backends. Thanks!